### PR TITLE
Urllib3 instead of urllib2 + Fix issue of defunct processes

### PR DESCRIPTION
**Pull request by anmolbabu**
tendrl-bug-id: Tendrl/performance-monitoring#44, Tendrl/performance-monitoring#45
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

File: tendrl/performance_monitoring/time_series_db/dbplugins/graphite.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class GraphitePlugin(TimeSeriesDBPlugin):
     def intialize(self):
         self.host = tendrl_ns.config.data['time_series_db_server']
         self.port = tendrl_ns.config.data['time_series_db_port']
+        self.http = urllib3.PoolManager()
```
Comment (shtripat) [inline tendrl/performance_monitoring/time_series_db/dbplugins/graphite.py]: We need changes in spec file as well for this?

Comment (anmolbabu) [inline tendrl/performance_monitoring/time_series_db/dbplugins/graphite.py]: Spec already had urllib3 :)

**Comment by anmolbabu** [comment]
@Tendrl/tendrl-core Please review

File: tendrl/performance_monitoring/aggregator/summary.py [hunk 2]
```diff
@@ -171,15 +171,9 @@ def calculate_host_summary(self, node):
...
-        procs = []
         nodes = tendrl_ns.central_store_thread.get_node_ids()
         for node in nodes:
-            proc = multiprocessing.Process(target=self.calculate_host_summary, args=(node,))
-            procs.append(proc)
-            proc.start()
-        for proc in procs:
-            proc.join()
-            #self.calculate_host_summary(node)
+            gevent.spawn(self.calculate_host_summary, node)
```
Comment (r0h4n) [inline tendrl/performance_monitoring/aggregator/summary.py]: I hope there arent any global/shared variables being written in self.calculate_host_summary

**Pull request merged**